app.js

```diff
@@ -52,4 +52,254 @@ document.addEventListener('DOMContentLoaded', () => {
+  const html = document.querySelector('#l-1');
+  const bootstrap = document.querySelector('#l-2');
+  const rails = document.querySelector('#l-3');
+  const project = document.querySelector('.pro-image');
+  const text = document.querySelector('.short-summary');
+  const live = document.querySelector('.pop-button-l');
+  const source = document.querySelector('.pop-button-s');
+
+  // Popup Window
+  const projectItems = [{
+    title: 'Multi Post Stories',
+    close: 'x',
+    skillTag: ['html', 'Bootstrap', 'Ruby on Rails'],
+    imgSrc: './image/portfolio-snapshot.png',
+    summary: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+    liveButton: 'See Live',
+    liveImg: './image/live-img.png',
+    sourceButton: 'See Source',
+    sourceImg: './img/github.png',
+  },
+  {
+    title: 'Descriptive Projects',
+    close: 'x',
+    skillTag: ['html', 'Bootstrap', 'Ruby on Rails'],
+    imgSrc: './image/portfolio-snapshot.png',
+    summary: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+    liveButton: 'See Live',
+    liveImg: './image/live-img.png',
+    sourceButton: 'See Source',
+    sourceImg: './img/github.png',
+  },
+  {
+    title: 'Multi Post Stories',
+    close: 'x',
+    skillTag: ['html', 'Bootstrap', 'Ruby on Rails'],
+    imgSrc: './image/portfolio-snapshot.png',
+    summary: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+    liveButton: 'See Live',
+    liveImg: './image/live-img.png',
+    sourceButton: 'See Source',
+    sourceImg: './img/github.png',
+  },
+  {
+    title: 'Too much Projects',
+    close: 'x',
+    skillTag: ['html', 'Bootstrap', 'Ruby on Rails'],
```

Comment:
- [ ] As mentioned by the previous reviewer your design needs to match as much as possible the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471), please ensure that your popup has the same size of skillTag, fonts, padding, and margin, also your closing button :grin: :grin: 

Your design 
![Screenshot from 2022-07-29 00-50-38](https://user-images.githubusercontent.com/74501523/181651417-80dafee7-3fb4-45d0-9893-847dae92c5c7.png)

Figma design
![Screenshot from 2022-07-29 00-53-30](https://user-images.githubusercontent.com/74501523/181651483-186b7751-8c34-4b66-947a-aa67cd6d1b32.png)